### PR TITLE
Fix invalid pip syntax, use double quotes for portable version (zsh/windows)

### PR DESCRIPTION
**Pull request by bmerkle**
pip install command with single quotes results in the following error:

## Why are these changes needed?
because it results in a pip syntax error

pip install command with single quotes results in the following error:

```
(venv) C:\work\microsoft\autogen>pip install 'autogen-agentchat==0.4.0.dev8' 'autogen-ext[openai]==0.4.0.dev8'

ERROR: Invalid requirement: "'autogen-agentchat==0.4.0.dev8'": Expected package name at the start of dependency specifier
    'autogen-agentchat==0.4.0.dev8'
    ^

```

## Related issue number
as this is a trivial change, i did not create a separate issue

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by ekzhu** [comment]
Ironically this will break usage in ZSH...

Let's use double quotes.

**Comment by bmerkle** [comment]
yes, I will do this. 
Lets use double quotes

File: python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/agent-and-agent-runtime.ipynb
Comment (ekzhu) [inline python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/agent-and-agent-runtime.ipynb:1]: This file shouldn't have been changed.

File: python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/distributed-agent-runtime.ipynb
Comment (ekzhu) [inline python/packages/autogen-core/docs/src/user-guide/core-user-guide/framework/distributed-agent-runtime.ipynb:1]: This file shouldn't have been changed except for updating the installation instructions.

File: python/packages/autogen-core/docs/src/user-guide/core-user-guide/cookbook/local-llms-ollama-litellm.ipynb
Comment (ekzhu) [inline python/packages/autogen-core/docs/src/user-guide/core-user-guide/cookbook/local-llms-ollama-litellm.ipynb:1]: This file shouldn't have been changed except for updating the installation instructions.

File: python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/tutorial/models.ipynb
Comment (ekzhu) [inline python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/tutorial/models.ipynb:1]: This file shouldn't have been changed except for updating the installation instructions.

File: python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/quickstart.ipynb
Comment (ekzhu) [inline python/packages/autogen-core/docs/src/user-guide/agentchat-user-guide/quickstart.ipynb:1]: This file shouldn't have been changed except for updating the installation instructions.

**Comment by bmerkle** [comment]
I read your comments but IMO these are normal shell scripts, just inside a jupyter notebook cell, so we should update this also.
I have verified that the notebook runs, so there is no error, and it should be consistent between docs and code.

We should maybe consider even to use "%pip install ..." instead of "pip install ...." in the ipynb files (with percentage) as this is a good practice (to enforce installation into the venv envionment). 

what do you think ?

Reference: 
https://community.databricks.com/t5/data-engineering/whats-the-difference-between-magic-commands-pip-and-sh-pip/td-p/21618
https://stackoverflow.com/questions/65547021/difference-between-pip-and-pip-in-python-jupyter-notebook-and-colab

**Comment by bmerkle** [comment]
> > I have verified that the notebook runs, so there is no error, and it should be consistent between docs and code.
> 
> My comment was that I saw the outputs of the notebooks have been cleared -- we should keep them
ok, sorry i misunderstood you. I have a commit hook for jupyter file to clean the output :-) 
should I add the output again and commit again ?

> > We should maybe consider even to use "%pip install ..." instead of "pip install ...." in the ipynb files (with percentage) as this is a good practice (to enforce installation into the venv envionment).
> 
> I agree.
cool, so i would change this to %pip install

**Comment by ekzhu** [comment]
No worries. Yes please reset changes on those files and apply double quotes again.

**Comment by jackgerrits** [comment]
I have resolved the issue here: #4760

**Pull request closed without merging**